app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt

```diff
@@ -7,10 +7,10 @@ import java.util.*
 
 class MainActivity : AppCompatActivity() {
 
-    val calc = Calculator()
-    val numstack: Stack<Double> = Stack()
-    var lastOperator: Char? = null
-    var last: Double? = null
+    private val calc = Calculator()
+    private val digitStack: Stack<Double> = Stack()
+    private var lastOperator: Char? = null
+    private var last: Double? = null
```

Comment:
non descriptive variable. Please meet standards of company.

Reply:
SCS Coding Standards too harsh... but you're right pretty non descriptive